### PR TITLE
[docs/website] Fix floating navbar anchor issue

### PR DESCRIPTION
**Pull request by charlieegan3**
Fixes https://github.com/open-policy-agent/opa/issues/5774

### CODE REVIEW

File: docs/website/assets/sass/docs.sass [hunk 1]
```diff
@@ -1,5 +1,14 @@
 $panel-header-logo-width: 85%
 
+// this will add space before the target element so that the floating navbar
+// doesn't cover it when following an anchor link
+\:target::before
+  +desktop
+    content: ""
+    display: block
+    height: 3.75rem // height of the navbar + 0.5rem
```
Comment (srenatus) [inline docs/website/assets/sass/docs.sass:9]: I think I'd increase this a bit...

image

Feels like some extra space wouldn't hurt. But it's not a blocker, this is so much better than before. Also just a matter of preference, I guess.

**Review by srenatus — DISMISSED** [review]
Thanks! 👏

File: docs/website/assets/sass/docs.sass (hunk 1 above)
Comment (charlieegan3) [reply docs/website/assets/sass/docs.sass]: I've increased it to 4.25 now. I also realised that the margin wasn't needed.

One thing to note is that this space is inserted into the page so I was trying to keep it fairly small.

https://user-images.githubusercontent.com/1774239/226322111-8681c45b-37e0-4c14-9ec1-4c298bebfc58.mov

Comment (charlieegan3) [reply docs/website/assets/sass/docs.sass]: (I'm not sure we have any other option as the dials on the markdown parser are limited)

**Review by ashutosh-narkar — APPROVED** [review]

**Review by srenatus — APPROVED** [review]
Thanks

**Pull request merged**